content/hardware/10.mega/shields/giga-display-shield/tutorials/13.appwizard-guide/content.md

```diff
@@ -0,0 +1,135 @@
+---
+title: GIGA Display Shield AppWizard Guide
+description: 'Learn how to use Segger AppWizard with the GIGA Display Shield.'
+author: Benjamin Dannegård
+tags: [Display, appwizard, segger, GUI]
+---
+
+## Introduction
```

Comment:
Would be nice to have a thumbnail here with the Giga Display showing a UI.

Reply:
idem about: hello @mcmchris please use an already done image on the tutorial to use it as initial image or a quick small trick so we can move this to publish! :pray:

Reply:
Done!